projects/components/select/src/select.component.spec.ts

```diff
@@ -201,16 +204,8 @@ describe('PsSelectComponent', () => {
     // expect(component.emittedValues).toEqual([undefined]);

```

Comment:
remove?

Reply:
no, this is only commented out until the bug in MatSelect is fixed